tests/test_fdb.py

```diff
@@ -122,10 +86,114 @@ def test_FDBAddedAfterMemberCreated(dvs):
+    rc = dvs.servers[16].runcmd("ping -c 1 6.6.6.7")
+    assert rc == 0
+
+    # Get mapping between interface name and its bridge port_id
+    iface_2_bridge_port_id = get_map_iface_bridge_port_id(dvs.adb, dvs)
+
+    # check that the FDB entries were inserted into ASIC DB
+    ok, extra = dvs.is_fdb_entry_exists(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_FDB_ENTRY",
+                    [],
+                    [("SAI_FDB_ENTRY_ATTR_TYPE", "SAI_FDB_ENTRY_TYPE_DYNAMIC"),
+                     ("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID", iface_2_bridge_port_id["Ethernet64"]),
+                    ]
+    )
+    assert ok, str(extra)
+    ok, extra = dvs.is_fdb_entry_exists(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_FDB_ENTRY",
+                    [],
+                    [("SAI_FDB_ENTRY_ATTR_TYPE", "SAI_FDB_ENTRY_TYPE_DYNAMIC"),
+                     ("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID", iface_2_bridge_port_id["Ethernet68"]),
+                    ]
+    )
+    assert ok, str(extra)
+
+    # check that the FDB entries were inserted into State DB
+    ok, extra = dvs.is_table_entry_exists(dvs.sdb, "FDB_TABLE",
+                    "Vlan6:.*",
+                    [("port", "Ethernet64"),
+                     ("type", "dynamic"),
+                    ]
+    )
+    assert ok, str(extra)
+    ok, extra = dvs.is_table_entry_exists(dvs.sdb, "FDB_TABLE",
+                    "Vlan6:*",
+                    [("port", "Ethernet68"),
+                     ("type", "dynamic"),
+                    ]
+    )
+    assert ok, str(extra)
+
+    # enable warm restart
+    # TODO: use cfg command to config it
+    create_entry_tbl(
+        dvs.cdb,
+        swsscommon.CFG_WARM_RESTART_TABLE_NAME, "swss",
+        [
+            ("enable", "true"),
+        ]
+    )
+
+    try:
+        # restart orchagent
+        dvs.runcmd(['sh', '-c', 'supervisorctl restart orchagent'])
+        time.sleep(2)
+
+        # Get mapping between interface name and its bridge port_id
+        # Note: they are changed
+        iface_2_bridge_port_id = get_map_iface_bridge_port_id(dvs.adb, dvs)
+
+        # check that the FDB entries were inserted into ASIC DB
+        ok, extra = dvs.is_fdb_entry_exists(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_FDB_ENTRY",
+                        [],
+                        [("SAI_FDB_ENTRY_ATTR_TYPE", "SAI_FDB_ENTRY_TYPE_DYNAMIC"),
+                         ("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID", iface_2_bridge_port_id["Ethernet64"]),
+                        ]
+        )
+        assert ok, str(extra)
+        ok, extra = dvs.is_fdb_entry_exists(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_FDB_ENTRY",
+                        [],
+                        [("SAI_FDB_ENTRY_ATTR_TYPE", "SAI_FDB_ENTRY_TYPE_DYNAMIC"),
+                         ("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID", iface_2_bridge_port_id["Ethernet68"]),
+                        ]
+        )
+        assert ok, str(extra)
+
+    finally:
+        # disable warm restart
+        # TODO: use cfg command to config it
```

Comment:
"config warm_restart enable swss" is available.

Reply:
I am following tests in test_warm_reboot.py. I will update later if those tests updated.

---
In reply to: [219020740](https://github.com/Azure/sonic-swss/pull/615#discussion_r219020740) [](ancestors = 219020740)